src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/ClassWithChainedMethods.java

```diff
@@ -24,7 +24,7 @@ public static void main(String[] args) { //indent:2 exp:2
     .length(); //indent:4 exp:>=8 warn
 
     int length2 =  //indent:4 exp:4
-    new ClassWithChainedMethods("param1","param2") //indent:4 exp:>=8 warn
+    new ClassWithChainedMethods("param1","param2") //indent:4 exp:4
```

Comment:
it should be violation.

Reply:
Yes, it is suspicious. This file when I checked against the same config, thowed 17 errror (including above). But this is showing only 3 errors. 
Maybe it is only catching `warns` and not `errors` and since code for checking `new` is entirely  moved to its handler, The handler throws errors and doesn't being caught under the `warns` categories.
See the test logic: https://github.com/checkstyle/checkstyle/blob/820e08ba250eca642319f1ec74fef08848d784eb/src/it/java/com/google/checkstyle/test/chapter4formatting/rule4841indentation/IndentationTest.java#L139-L152

Reply:
see `getLinesWithWarn(filePath); ` on line 150.

Reply:
please make comment `//indent:4 exp:8 warn`

we did not make it enforced to keep comment aligned with actual violation.
It is enforced in `it` folder, but not in `test` folder. It is one of future GSoC projects to make our UTs be more behavior focused and comments in Input actually define expected violation.

Reply:
Corrected!, actually the checkConfig here is only `Indentation`, and I thought it is following the default indentation profile, and I checked it against cli on terminal and everything was fine.

But later on further debugging, founded that it was just another config with different properties and indentation attributes. 
Checked it only cli, and got the same result, then further found  that the problem was caused be chained method calls where the new is not the direct child of assign which was not covered in the current code logic.
Now Corrected!